src/objects.js

```diff
@@ -1,41 +1,59 @@
 const createPerson = (name, age) => {
-  // your code here
+return {
+  name: name,
+  age: age,
+
+};
 };
 
 const getName = object => {
-  // your code here
+return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+return object.hasOwnProperty;
 };
 
 const isOver65 = person => {
-  // your code here
+if (person.age > 65) {
+  return true;
+} else {
+  return false;
+}
 };
 
 const getAges = people => {
-  // your code here
+const arrayAges = people.map(person => person.age);
+return arrayAges;
 };
 
 const findByName = (name, people) => {
-  // your code here
+const person = people.find(person => person.name === name);
+return person;
 };
 
 const findHondas = cars => {
-  // your code here
+const hondas = cars.filter(car => car.manufacturer === 'Honda');
+return hondas;
 };
 
 const averageAge = people => {
-  // your code here
+let totalAge = 0;
+for (i=0; i < people.length; i++) {
+  totalAge += people[i].age;
+
+return totalAge/people.length;
+}
 };
```

Comment:
this function isn't working because you have the closing bracket for the `for` loop after your `return`, which means on the first loop you are returning the result, which stops any more code in the function being run. To fix this, simply move the closing bracket on line `50` to line `48`, i.e.
```
const averageAge = people => {
  let totalAge = 0;
  for (let i = 0; i < people.length; i += 1) {
    totalAge += people[i].age;
  }

  return totalAge / people.length;
};
```